### DISCUSSION TITLE
Welcome to pyvideotrans Discussions!

### DISCUSSION THREAD

**Discussion by jianchang512** [jianchang512/pyvideotrans#23] [Announcements]
[Discord group](https://discord.gg/evkPeKJddD)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ManDoe-new** [comment]
Romanian Output Language needed, have installed Edge-TTS, but I can`t select any more languages as the initial once.

This is what ChatGPT told me, but I can not Choose any Settings-Item to Apply edge-TTS, but Edge-TTS is availabe in Dubb Channel.
What can I do?

I became from the AI-Support many feedback, but that does not fit to my installation. I have the following Installation with the menue bar above, and I do not have the Menu Translation to make konfigurations there:
<img width="1821" height="500" alt="grafik" src="https://github.com/user-attachments/assets/377d1fd2-50a7-4ab1-8fc9-17655db40144" />

How can I get Romanian as a choosable Output language?

Thanks for your help.

**Comment by jianchang512** [comment]
<img width="737" height="597" alt="image" src="https://github.com/user-attachments/assets/173968a3-d7db-4e2a-bea5-5ec34eea7163" />

only there can use
